eo-phi-normalizer/src/Language/EO/Phi/Normalize.hs

```diff
@@ -93,6 +76,7 @@ peelObject = \case
   ThisObject -> PeeledObject HeadThis []
   Termination -> PeeledObject HeadTermination []
   MetaObject _ -> PeeledObject HeadTermination []
+  MetaFunction _ _ -> error "To be honest, I'm not sure what should be here"
```

Comment:
Can a meta function be peeled somehow?

Reply:
Both this and previous lines should be updated, but in a separate PR, one that introduces meta variables and meta functions in the syntax for peeled objects. This is of low priority for now.